src/serverMiddleware/routes.js

```diff
@@ -7,10 +7,10 @@
 
 module.exports = [
 	`^/administration/datasources?`,
-	// `^/administration/students/?$`,
-	// `^/administration/students/new/?$`,
-	// `^/administration/teachers/?$`,
-	// `^/administration/teachers/new/?$`,
+	`^/administration/students/?$`,
+	`^/administration/students/new/?$`,
+	`^/administration/teachers/?$`,
+	`^/administration/teachers/new/?$`,
```

Comment:
should not be part of this PR. We will enable the routes for production here, when all the features are finished. Please do not commit changes in this file till then.
```suggestion
	// `^/administration/students/?$`,
	// `^/administration/students/new/?$`,
	// `^/administration/teachers/?$`,
	// `^/administration/teachers/new/?$`,
```

Reply:
sure ;)